exporters/jaeger-thrift/src/main/java/io/opentelemetry/exporter/jaeger/thrift/Adapter.java

```diff
@@ -78,8 +80,21 @@ static Span toJaeger(SpanData span) {
         TimeUnit.NANOSECONDS.toMicros(span.getEndEpochNanos() - span.getStartEpochNanos()));
 
     List<Tag> tags = toTags(span.getAttributes());
+    int numberOfAttributes = span.getAttributes().size();
```

Comment:
Possible minor readability improvement throughout

```java
int droppedOf* = size() - count();
if (droppedOf != 0) {
}
```

Reads the same as what we want, if there are dropped, record it

Reply:
that's totally fair; I can make that change tomorrow.

Reply:
done!